### PR TITLE
fix Hardtanh argument names to be consistent between functional and Module

### PR DESCRIPTION
**Pull request by soumith**
Fixes https://github.com/pytorch/pytorch/issues/1891

### CODE REVIEW

File: torch/nn/modules/activation.py [hunk 1]
```diff
@@ -126,10 +126,13 @@ class Hardtanh(Module):
...
     """
 
-    def __init__(self, min_value=-1, max_value=1, inplace=False):
+    def __init__(self, min_val=-1, max_val=1, inplace=False, min_value=None, max_value=None):
         super(Hardtanh, self).__init__()
-        self.min_val = min_value
-        self.max_val = max_value
+        self.min_val = min_val
+        self.max_val = max_val
+        assert min_value is None, "keyword argument min_value has been renamed to min_val"
```
Comment (alykhantejani) [inline torch/nn/modules/activation.py]: Do we rather want to add a deprecation warning and just do 
```
if min_value is not None:
    self.min_val = min_value
```
and then fully remove `min_value` and `max_value` in v0.2.x? This way it wouldn't break any existing code unexpectedly?

Comment (soumith) [inline torch/nn/modules/activation.py]: not a bad idea. I'll do this.

File: torch/nn/modules/activation.py [hunk 2]
```diff
@@ -109,8 +109,8 @@ class Hardtanh(Module):
     The range of the linear region :math:`[-1, 1]` can be adjusted
 
     Args:
-        min_value: minimum value of the linear region range
-        max_value: maximum value of the linear region range
+        min_val: minimum value of the linear region range
+        max_val: maximum value of the linear region range
```
Comment (alykhantejani) [inline torch/nn/modules/activation.py]: Might be good to mention in the docstring that `max_value` and `min_value` kwargs should not be used?

Comment (soumith) [inline torch/nn/modules/activation.py]: mentioned and pushed to master. thanks for the review

**Comment by soumith** [comment]
pushed to master

**Pull request closed without merging**